apps/videos-admin/src/app/(dashboard)/videos/[videoId]/_VideoBibleCitation/VideoBibleCitation.tsx

```
@@ -0,0 +1,185 @@
+        enqueueSnackbar('Error updating bible citation order:', {
+          variant: 'error'
+        })
+        void refetch()
+      }
+    }
+  }
+
+  return (
+    <Section title="Bible Citations" variant="outlined">
+      {data == null || data.bibleCitations.length === 0 ? (
+        <Section.Fallback>No Bible citations</Section.Fallback>
```

ESLint

[error] disallow literal string: <Section.Fallback>No Bible citations</Section.Fallback>

(i18next/no-literal-string)